themes/inbeat/layouts/partials/site/newsletter-popup.html

```diff
@@ -21,7 +21,7 @@ <h1 class="logo-name">Growthnotes Newsletter</h1>
         </div>
         <div class="form-group">
           <input type="email" placeholder="Enter your email" v-model="email" />
-          <a v-if="!loading" @click.prevent="handleSubscription" href="#" class="cta" title="Try it">
```

Comment:
Detail but this should be a `button` not an `a`. You can reset the button style if you don't want its uglyness!